symfit/core/argument.py

```diff
@@ -115,13 +120,17 @@ def __init__(self, name=None, value=1.0, min=None, max=None, fixed=False, **assu
         self.value = value
         self.fixed = fixed
 
-        if min is not None and max is not None and min > max:
-            if not self.fixed:
+        if min is not None and max is not None:
+            test = min > max
+            if isinstance(test, np.ndarray):
+                test = test.any()
+
+            if test and not self.fixed:
```

Comment:
```suggestion
        if min is not None and max is not None:
            if np.any(min > max) and not self.fixed:
```

`np.any` does what you want if you pass it plain bools.